lib/ActivityStreams.js

```diff
@@ -5,14 +5,17 @@
 const {Cu} = require("chrome");
 const {data} = require("sdk/self");
 const {PageMod} = require("sdk/page-mod");
+const {setTimeout} = require("sdk/timers");
 const {ActionButton} = require("sdk/ui/button/action");
 const tabs = require("sdk/tabs");
 const {PlacesProvider} = require("lib/PlacesProvider");
 const {SearchProvider} = require("lib/SearchProvider");
 const {TabTracker} = require("lib/TabTracker");
+const {PreviewProvider} = require("lib/PreviewProvider");
 const {TelemetrySender} = require("lib/TelemetrySender");
 const {AppURLHider} = require("lib/AppURLHider");
 const am = require("content-src/actions/action-manager");
+const PRIME_CACHE_TIMEOUT = 216000; // every 6 hours, prime the cache
```

Comment:
How about making that an option passed to ActivityStreams? That would make testing easier